superset-frontend/src/components/Chart/ChartRenderer.jsx

```diff
@@ -83,8 +82,10 @@ class ChartRenderer extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      showContextMenu:
+        props.source === 'dashboard' &&
```

Comment:
Do we have a better typescript definition for this to avoid using plain strings?

Reply:
Neither this component or the one that sets the prop is written in TS, but I was able to add an external module with an enum definition that I import and use.